src/main/java/com/google/sps/data/UserRepository.java

```diff
@@ -15,4 +15,6 @@
 
   //creates a User from 'googleUser' if one doesn't exist, otherwise returns existing User with same ID
   public User getUser(com.google.appengine.api.users.User googleUser);
+
+  public void addFakeMentors();
```

Comment:
Maybe add a note that addFakeMentors is for testing / demoing the MVP?

Reply:
Done!